src/deprecated.jl

```diff
@@ -9,3 +9,17 @@
 @deprecate CategoricalVector(::Type{T}, m::Integer; ordered=false) where {T} CategoricalVector{T}(m, ordered=ordered)
 
 @deprecate CategoricalMatrix(::Type{T}, m::Int, n::Int; ordered=false) where {T} CategoricalMatrix{T}(m, n, ordered=ordered)
+
+function Base.convert(::Type{Nullable{S}}, x::CategoricalValue{Nullable}) where S
+    Base.depwarn("The Nullable type is deprecated. Use Missing instead.", :convert)
```

Comment:
These methods only existed to fix ambiguities. I'd just put them under a `VERSION <=` check. We don't really care about `Nullable` in this package now.